assembly/src/parser/grammar.lalrpop

```diff
@@ -397,7 +397,7 @@ MacroInst: SmallOpsVec = {
 #[inline]
 Inst: Instruction = {
     AdviceInjector,
-    Assert,
+    MaybeWithError,
```

Comment:
*Nit:* I would name this something a bit more precise, perhaps `InstWithErrorCode`? Most other instances of `Maybe*`  are macros which can be applied to a non-terminal, but that isn't the case here - the others are more precise about what is being parsed, so less ambiguous at first glance.